src/lib/components/Modal/CollectionPopup.svelte

```diff
@@ -106,26 +106,26 @@
     class="d-flex align-items-center justify-content-end gap-3 mt-1 mb-0 rounded"
     style="font-size: 16px;"
   >
-    <CustomButton
+    <Button
       disable={deleteLoader}
-      text={"Cancel"}
-      fontSize={14}
+      title={"Cancel"}
+      textStyleProp={"font-size: 14px"}
       type={"dark"}
-      loader={false}
       onClick={() => {
         closePopup(false);
       }}
     />
 
-    <CustomButton
+    <Button
       disable={deleteLoader}
-      text={"Delete"}
-      fontSize={14}
+      title={"Delete"}
+      textStyleProp={"font-size: 14px"}

```

Comment:
same here for font size

Reply:
Resolved, using rem from global css for 12 and 14 pixel and fs-6 for 16px.